### PR TITLE
Implement refresh token to switch video manager

### PR DESCRIPTION
**Pull request by kenloburn**

### CODE REVIEW

**Review by robertszeker — APPROVED** [review]

File: src/MovingImage/Client/VMPro/Manager/TokenManager.php [hunk 1]
```diff
@@ -108,14 +107,39 @@ protected function createNewTokens()
...
+            $data = \json_decode($response->getBody(), true);
+            $logger->debug('Successfully retrieved new access token', $data);
+
+            return new Token(
+                $data['accessToken'],
+                $this->tokenExtractor->extract($data['accessToken']),
+                $videoManagerId
+            );
+        } catch (\Exception $e) {
+            throw $e;
```
Comment (omid) [inline src/MovingImage/Client/VMPro/Manager/TokenManager.php]: Did you do this on purpose?

Comment (robertszeker) [reply src/MovingImage/Client/VMPro/Manager/TokenManager.php]: The same error handling is also in createNewTokens(), but there is a ToDo in the function comment. So I guess we will add the error handling later for this class.

Comment (omid) [reply src/MovingImage/Client/VMPro/Manager/TokenManager.php]: It doesn't make sense to not to write an ugly code and write a bad code for later to make it good. :D
But never mind :)

**Review by omid — APPROVED** [review]

File: src/MovingImage/Client/VMPro/Manager/TokenManager.php (hunk 1 above)
Comment (kenloburn) [reply src/MovingImage/Client/VMPro/Manager/TokenManager.php]: I'm removing this no-op error handling until we implement scoped exceptions at a later time - pushing a change in a minute

**Pull request merged**